assets/src/common/components/pre-publish-panel.js

```diff
@@ -1,73 +1,26 @@
-/**
- * External dependencies
- */
-import PropTypes from 'prop-types';
-
 /**
  * WordPress dependencies
  */
-import { Notice } from '@wordpress/components';
+import { PostFeaturedImage } from '@wordpress/editor';
 import { PluginPrePublishPanel } from '@wordpress/edit-post';
-import { withSelect } from '@wordpress/data';
 import { __ } from '@wordpress/i18n';
 
 /**
- * Internal dependencies
- */
-import { validateFeaturedImage } from '../helpers';
-
-/**
- * Conditionally adds a notice to the pre-publish panel for the featured image.
+ * Adds a notice to the pre-publish panel for the featured image.
```

Comment:
The docblock for this component will also need to be updated to reflect these changes.

```suggestion
 * Adds a pre-publish panel containing the featured image selection component.
 *
 * Note: The `PostFeaturedImage` component would have already been filtered to include
 * any notices for the featured image so there is no need to recreate them here.
```